src/view/cards/Certification/__tests__/Certification.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen } from 'utils/testing';
+import { mockCerts } from 'services/api/__mocks__';
+import { CertificationCard } from '../Certification';
+
+describe('<Certificates />', () => {
```

Comment:
```suggestion
describe('<CertificationCard />', () => {
```

This one is a little weird because the file is just named `Certification.tsx`. However, I've been trying to use the actual component name in the describe, so it should be `<CertificationCard />`